app/code/Magento/CacheInvalidate/Model/PurgeCache.php

```diff
@@ -39,39 +39,46 @@ class PurgeCache
      *
      * @var int
      */
-    private $requestSize = 7680;
+    private $requestSize;
 
     /**
      * Constructor
      *
      * @param \Magento\PageCache\Model\Cache\Server $cacheServer
      * @param \Magento\CacheInvalidate\Model\SocketFactory $socketAdapterFactory
      * @param InvalidateLogger $logger
+     * @param int $maxHeaderSize
      */
     public function __construct(
         \Magento\PageCache\Model\Cache\Server $cacheServer,
         \Magento\CacheInvalidate\Model\SocketFactory $socketAdapterFactory,
-        InvalidateLogger $logger
+        InvalidateLogger $logger,
+        int $maxHeaderSize = 7680
     ) {
         $this->cacheServer = $cacheServer;
         $this->socketAdapterFactory = $socketAdapterFactory;
         $this->logger = $logger;
+        $this->requestSize = $maxHeaderSize;
```

Comment:
I think it's better to have single name for one thing. Can we name it as `requestSize` or `maxHeaderSize` in all places?